plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -123,6 +125,11 @@
         extends AbstractTrinoCatalog
 {
     private static final Logger LOG = Logger.get(TrinoGlueCatalog.class);
+    private static final RetryPolicy<?> CONCURRENT_MODIFICATION_EXCEPTION_RETRY_POLICY = RetryPolicy.builder()
```

Comment:
The Iceberg code will already retry on ConcurrentModificationExceptions. I think you meant to make this change in `GlueHiveMetastore`?

Reply:
Sorry. I was confused in the morning :( . I have to add anyway a test to verify the changes on Delta.